src/components/Person.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age !== 0 ? (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    ) : (
+      null
+    )}
```

Comment:
use logical operator (&&)

Reply:
I see '0' in the browser instead of just an empty line when I use && and a default value of age is 0. So I change a default value of age to null and add It to propTypes. I'm not sure that I'm doing It correctly